lib/manager/components/transform/ReplaceFileFromString.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import React from 'react'
+import {Button} from 'react-bootstrap'
+
+import Transform from './Transform'
+
+import type { ReplaceFileFromStringFields } from '../../../types'
+import type { TransformProps } from './Transform'
+
+/**
+ * Component that renders fields for ReplaceFileFromStringTransformation.
+ */
+export default class ReplaceFileFromString extends Transform<ReplaceFileFromStringFields> {
+  constructor (props: TransformProps<ReplaceFileFromStringFields>) {
```

Comment:
I agree with @evansiroky about avoiding inheritance... it just takes us away from patterns that are used everywhere else in our apps and makes it tough for the reader to switch contexts.

Reply:
Addressed in https://github.com/ibi-group/datatools-ui/pull/649/commits/54f1aac3573cc5af4575bd839afb116708e626e8